esti/lakectl_doctor_test.go

```diff
@@ -1,18 +1,23 @@
 package esti
 
 import (
+	"github.com/stretchr/testify/require"
+	"net/url"
 	"testing"
 
 	"github.com/spf13/viper"
 )
 
-func TestDoctor(t *testing.T) {
+func TestLakectlDoctor(t *testing.T) {
 	SkipTestIfAskedTo(t)
 	accessKeyID := viper.GetString("access_key_id")
 	secretAccessKey := viper.GetString("secret_access_key")
 	endPointURL := viper.GetString("endpoint_url") + "/api/v1"
+	u, err := url.Parse(endpointURL)
+	require.NoError(t, err)
 	vars := map[string]string{
 		"LAKEFS_ENDPOINT": endPointURL,
+		"HOST":            u.Host,
```

Comment:
LAKEFS_HOST or LAKEFS_HOSTPORT?

Reply:
The url struct member is called `Host` so why should we change the name?